docusaurus.config.js

```diff
@@ -9,8 +9,8 @@ const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 (module.exports = {
   title: "Flashbots",
   tagline: "Research respository of Flashbots",
-  baseUrl: process.env.REACT_APP_BASE_URL,
-  url: process.env.REACT_APP_TARGET_URL,
+  baseUrl: "/",
```

Comment:
i don't think you should change this file? it makes the gh checks fail too but also last post didn't need this change and worked just fine

Reply:
Yes this is some local testing what I forgot to delete